frameworks/cassandra/tests/conftest.py

```diff
@@ -1,37 +1,21 @@
-import json
-import logging
-
 import pytest
+import sdk_plan
 import sdk_repository
 import sdk_security
-import sdk_utils
-import shakedown
 
-log = logging.getLogger(__name__)
+SERVICE_NAME = 'cassandra'
```

Comment:
We usually define `SERVICE_NAME` in [`config.py`](https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/tests/config.py#L16) for the services so perhaps use/import that for convergence's sake?

Reply:
i tried consuming from there directly, but `import config` failed

Reply:
ah, might need to specify directory i.e. `from tests import config` as in https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/tests/test_sanity.py#L15

Reply:
aha, yep, forgot about that